index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Using the same "brand" as the types for `htmlbars-inline-precompile` for
+// backwards compatibility. The actual value of the brand doesn't matter; it is
+// only important that it (a) is distinct and (b) interoperates with existing
+// uses of the `hbs` export from `htmlbars-inline-precompile`.
```

Comment:
Can you link to the "source of truth" for this brand in this comment?

Reply:
Excellent suggestion; will do.

Reply:
Done in 99738f2